### ISSUE TITLE
suggestion to add an alternative `l`

### ISSUE THREAD

**Issue by disconsis** [subframe7536/maple-font#159]
Hey, thank you so much for your work on this! It looks amazing.

This is more of a pet peeve than anything else, but I would love for there to be an alternative `l` similar to the original `i` (currently it's more similar to the alternative `i`)

This is somewhat what I was thinking:
![image](https://github.com/subframe7536/maple-font/assets/18366180/d2bb3a64-e6a3-4d96-9cc9-6ac4aa79a193)

I did this in paint by pasting the bottom of the original `i` on the `l` (I'm bad at font stuff 😢)

**Comment by subframe7536** [comment]
Sure, I will add it. But it might implemented in another stylesets

**Comment by disconsis** [comment]
Thank you!

**Comment by subframe7536** [comment]
preview
![recording](https://github.com/subframe7536/maple-font/assets/78338239/954fc885-bc03-4af3-9e9a-94288c270aaa)

**Comment by disconsis** [comment]
Thank you so much for your work!

**Comment by disconsis** [comment]
Could you give some hints on which branch/tag to use and features to enable to try this?

**Comment by subframe7536** [comment]
`cv03`，as shown at the right part of gif

only available in v7, which have no stable release

**Comment by disconsis** [comment]
Ah okay. I'm not very familiar with font tools, so I think I'll just wait for the stable release :)

**Comment by subframe7536** [comment]
temporary reopen it, close after V7 stable release

**Comment by 3T1C** [comment]
Could you please make another version that removes the top of the letter L like in this image? Thank you for your hard work!

![image](https://github.com/subframe7536/maple-font/assets/49277107/4001b524-dd7f-42c5-8556-32736434379d)

![image](https://github.com/subframe7536/maple-font/assets/49277107/6323395f-419b-4996-922e-aa6a338c4746)

**Comment by subframe7536** [comment]
I think this style is more suitable for sans instead of monospace.

Could you please provide some code example image includes this style of l?

**Comment by 3T1C** [comment]
I agree that this style is more suitable for sans rather than monospace. I've only found the 'l' shape that I like in fonts like Monoid or Terminal One. It's just a personal preference, but if it's possible to create a version with the top of the 'l' removed, that would be great. If not, that's okay too. Thanks again for creating such an amazing font.

**Comment by subframe7536** [comment]
@disconsis do you find that `1` is similar to `l` when enable `cv04`? I am considering to remove the bottom bar of `1` as well. Any thought or suggestion is appreciated.

**Comment by disconsis** [comment]
I think that makes a lot of sense. It is quite similar otherwise.

**Comment by subframe7536** [comment]
![recording](https://github.com/subframe7536/maple-font/assets/78338239/92af3410-2604-4ee4-b39c-179f277992af)
will release in v7-beta20

**Comment by subframe7536** [comment]
release in v7-beta20
